src/tests.c

```diff
@@ -1,12 +1,8 @@
 /*
  * This file contains unit tests for C-KZG-4844.
  */
-#include "common.c"
+#include "ckzg.c"
```

Comment:
IMO it's better to include the .c files that actually have the functions, instead of the mother file. I think it makes dependencies more explicit. No strong opinion.

Reply:
Because this is a test file for everything, I think it makes sense to include everything.